src/ClimbObc.c

```diff
@@ -73,6 +75,12 @@ static gps_initdata_t GpsInit = {
 		PTR_FROM_IDX(PINIDX_STACIE_C_IO1_P)
 };
 
+static thr_initdata_t ThrInit = {
+		LPC_UART1, /// WILL USE UART1 since it is on Y+ side, which according to doccumentation given to me should be used for RS485 thruster LPC_UART1
+		PTR_FROM_IDX(PINIDX_GPIO4_CP),
```

Comment:
Do you use this additional Pins for thruster controll. If so pls. comment here which pins with which functions. (init structure was copied from GPS and there where pps and enable pin...)